src/utils/checksum.ts

```diff
@@ -1,14 +1,16 @@
+// no type definitions but deprecated anyways, so:
+// @ts-ignore
 import {Md5} from 'ts-md5/dist/md5';
 import {recursiveObjectSort} from "./object";
 
 /**
  * Method for generating simple object checksums
  * @author Mike Reiche <mike.reiche@t-systems.com>
  * @param object
- * @param objects
+ * @param _objects
```

Comment:
Empty tag. But it's already deprecated so I just mention it to "build awareness". No need to act on it.